jax/_src/numpy/ufuncs.py

```diff
@@ -1990,16 +1990,18 @@ def remainder(x1: ArrayLike, x2: ArrayLike, /) -> Array:
 
   JAX implementation of :obj:`numpy.remainder`.
 
-  The remainder is computed by ``x1 - x2 * jnp.floor(x1 / x2)``.
-
   Args:
     x1: scalar or array. Specifies the dividend.
     x2: scalar or array. Specifies the divisor. ``x1`` and ``x2`` should either
       have same shape or be broadcast compatible.
 
   Returns:
     An array containing the remainder of element-wise division of ``x1`` by
-    ``x2`` with same dtype as inputs and sign as that of the elements of ``x2``.
+    ``x2`` with same sign as the elements of ``x2``. The output dtype is the type
+    promotion result of ``x1`` and ``x2`` upcast to inexact.
```

Comment:
Actually, I was incorrect here: it's not upcast to inexact. Honestly, looking at this now, I'd just remove the sentence about the output dtype here. We don't mention the output dtypes in other ufunc docstrings.

Reply:
Thanks! Removed the mention of output dtype.